client/src/components/Login/index.jsx

```diff
@@ -1,68 +1,61 @@
 import React, { Component } from 'react';
 import './style.css';
-import Logo from '../Logo';
+import Logo from '../Logo'
 
-
-export default class extends Component {
-  state = {
-    email: '',
-    password: '',
-    msg:''
-  };
-
-   
-  onSubmit = event => {
-    event.preventDefault();
-    console.log(event.target.email.value);
-    const requestInfo = {
-      emailValue: event.target.email.value,
-
-      passwordValue: event.target.password.value
+export default class Login extends Component {
+    state = {
+        emailValue:'',
+        passwordValue:'',
+        msg:''
     }
-    fetch('/login', {
-      method: 'POST',
-      body: JSON.stringify(requestInfo),
-      headers: {
-        'content-type': 'application/json'
-      }
-    })
-    .then(res=>res.json())
-    .then(({msg}) => {
-     this.setState({msg:'login successfully'});
-    });
-  }
-  render() {
-    return (
-      <div className="login">
-        <main className="login__main">
-          <section className="login__header">
-            <Logo className="login__logo" />
-            <span className="login__logo-text login__logo-text--white">UN</span>
-            <span className="login__logo-text login__logo-text--black">OPS</span>
-          </section>
-          <form onSubmit={this.onSubmit} className="login__form">
-            <label htmlFor="email" className="login__label">
-              <span className="login__label-text">
-                Email:
-                {' '}
-                <span className="required">*</span>
-              </span>
-              <input type="email" id="email" className="login__input" />
-            </label>
-            <label htmlFor="password" className="login__label">
-              <span className="login__label-text">
-                Password:
-                {' '}
-                <span className="required">*</span>
-              </span>
-              <input type="password" id="password" className="login__input" />
-            </label>
-            <input type="button" className="login__submit" value="Login" />
+    onSubmit = event => {
+        event.preventDefault();
+       this.setState({emailValue: event.target.email.value,passwordValue: event.target.password.value});
+        const {emailValue,passwordValue}=this.state;
```

Comment:
@alaabadra You could use event.target.email.value and event.target.password.value in your fetch instead of storing them in state right here (line 13) and then immediately getting them back out of state (line 14).
React can sometimes batch up state changes, as a performance optimization, which means that sometimes if you call setState and then on the next line call getState (or call a function on the next line which then calls setState), your new state might not yet be in this.state, so you can sometimes see bugs that come from things like that.